src/scripts/templates/active/Active default template.js

```diff
@@ -4,7 +4,14 @@
 // Note that new active scripts will initially be disabled
 // Right click the script in the Scripts tree and select "enable"  
 
-
+/**
+ * Scans a "node", i.e. an individual entry in the Site Tree.
```

Comment:
"Sites" instead?

Guess the first two lines of the file could be deleted, replaced by new docs?